src/aatoolbox/config/countryconfig.py

```diff
@@ -31,11 +31,53 @@ class CodABConfig(BaseModel):
     custom_layer_names: Optional[list]
 
 
+class FewsNetConfig(BaseModel):
+    """FEWS NET configuration.
+
+    Parameters
+    ----------
+    region_name: str
+        Name of the region the country belongs to. Needed to download the
+        regional FEWS NET data
+    """
+
+    # values dictionary gets build in order attributes are listed
+    # so first define the dict before the region_name
+    region_name_code_mapping: Dict[str, str] = {
```

Comment:
@turnerm I now added the list of region name - code combinations here and then also assigned the region code in the config. Does this make sense to you?

Reply:
perfect!!